locales/es.yaml

```diff
@@ -56,3 +56,4 @@ es:
   continue: continuar
   error_amount: Por favor, seleccione un valor que no sea menor que {{ min_amount }}.
   donate_now_amount: Donar {{ currency_symbol }}{ amount } ahora
+  other_ways_to_give:
```

Comment:
I would rather do it in the code than in this translation file since this will be display on Transifex.